ionic-app/components/Menu.jsx

```diff
@@ -0,0 +1,80 @@
+import { StatusBar, Style } from '@capacitor/status-bar';
+import {
+  IonContent,
+  IonHeader,
+  IonIcon,
+  IonItem,
+  IonLabel,
+  IonList,
+  IonMenu,
+  IonMenuToggle,
+  IonTitle,
+  IonToolbar,
+} from '@ionic/react';
+import { useEffect, useState } from 'react';
+import { cog, flash, list } from 'ionicons/icons';
+
+const pages = [
+  {
+    title: 'Feed',
+    icon: flash,
+    url: '/tabs/feed',
+  },
+  {
+    title: 'Lists',
+    icon: list,
+    url: '/tabs/lists',
+  },
+  {
+    title: 'Settings',
+    icon: cog,
+    url: '/tabs/settings',
+  },
+];
+
+const Menu = () => {
+  const [isDark, setIsDark] = useState(false);
+
+  const handleOpen = async () => {
+    try {
+      await StatusBar.setStyle({
+        style: isDark ? Style.Dark : Style.Light,
+      });
+    } catch {}
+  };
+  const handleClose = async () => {
+    try {
+      await StatusBar.setStyle({
+        style: isDark ? Style.Dark : Style.Light,
+      });
+    } catch {}
+  };
+
+  useEffect(() => {
+    setIsDark(window.matchMedia('(prefers-color-scheme: dark)').matches);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/toost-net/toost-cloud/pullRequest?prid=8079923)
```suggestion
    setIsDark(window.matchMedia("(prefers-color-scheme: dark)").matches);
```